### DISCUSSION TITLE
Nodejs pool controller vm

### DISCUSSION THREAD

**Discussion by sts098** [tagyoureit/nodejs-poolController#1008] [Q&A]
I have a several VSphere servers used for web, erp,etc for a small business.  Considering setting up a Linux vm to host nodejs-pool controller.  I currently have a raspberry pi 4 running atlas iot with their ilink3 board and temperature,ph and Orp probes.  Wondered if I could run  rely equipment manager locally on the pi4 and nodejs-pc on a vm.  Would offer me ability to backup with rest of system and should be quicker.  Not sure how nodejs-pc communicates with rem.  Would this be possible.  Any downsides?

**Comment by rstrouse** [comment]
Yes REM is designed to interact with the hardware and interface with njsPC and dashPanel using TCP out of process.

I don’t think there will be a performance boost that is noticeable though.
